documentation/examples/prometheus-nomad.yml

```diff
@@ -0,0 +1,17 @@
+# An example scrape configuration for running Prometheus with
+# Nomad build in service discovery.
+
+scrape_configs:
+  # Make Prometheus scrape itself for metrics.
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:9090"]
+
+  # Discover Node Exporter instances to scrape.
+  - job_name: 'nomad_sd'
+    nomad_sd_configs:
+      - server: 'http://localhost:4646'
+    relabel_configs:
+      # Only scrape targets that have a tag 'monitoring'.
```

Comment:
I feel like the example is not complete yet, and the comments are not accurate

Reply:
removed the comments, the example currently reflects the connection variables i have implemented sofar. When i add the missing parameters here. But this is a copy of the prometheus.yml that i use to test this new code